deploy/testnet/master-validator/init.sh

```diff
@@ -143,4 +154,4 @@ echo "delete faucet account if we have previously"
 $GRAVITY keys delete faucet --keyring-backend test -y &
 
 echo "Starting faucet based on validator account"
-faucet -cli-name=$GRAVITY -keyring-backend=test -mnemonic="$ONOMY_ORCHESTRATOR_MNEMONIC" &
\ No newline at end of file
+faucet -cli-name=$GRAVITY -keyring-backend=test -mnemonic="$ONOMY_ORCHESTRATOR_MNEMONIC" credit-amount=100000000  -max-credit=200000000 &
```

Comment:
Have you tested the faucet with those params?

Reply:
I am trying to test the faucet using the flag credit-amount=100,000,000 -max-credit=200,000,000
if I try for 200,000,001 goes fail but when trying call request 2,000,000,000 it goes successful.

I have raised the issue with that https://github.com/tendermint/faucet/issues/10